perf/benchmark/run_benchmark_job.sh

```diff
@@ -117,12 +117,12 @@ function setup_fortio_and_prometheus() {
 
 #TODO: add stackdriver filter
 function prerun_v2_nullvm() {
-  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v1.enabled=false,values.telemetry.v2.enabled=true,values.telemetry.v2.prometheus.enabled=true"
+  export SET_OVERLAY="--set values.telemetry.enabled=true --set values.telemetry.v1.enabled=false --set values.telemetry.v2.enabled=true --set values.telemetry.v2.prometheus.enabled=true"
```

Comment:
If as stated telemetryv2 is enbled in 1.5 and master by default, then it means currently our mixer_mode is running in telemetryv2. That's why my manual test runs got strange results. @mandarjog 
Yeah, we need to explicitly enable mixer when we are testing mixer_mode.